package.json

```diff
@@ -5,20 +5,23 @@
   "main": "dist/server.bundle.js",
   "scripts": {
     "bootstrap": "shx rm -rf dist && shx mkdir dist",
-    "start": "npm run build && npm-run-all --parallel server watch:**",
-    "build": "npm run build:config && npm run build:client && npm run build:server",
-    "build:server": "webpack --config webpack.server.config.js",
-    "build:client": "npm run bootstrap && npm run build:assets && npm run build:js && npm run build:scss && npm run build:static",
-    "build:assets": "cp -r assets dist",
+    "start": "npm run build && run-p server watch:**",
+    "build": "npm run bootstrap && run-s build:**",
```

Comment:
both `npm run` and `run-s` are already sequential, so you can turn this into `run-s bootstrap build:**`